apps/good_practice/types.py

```diff
@@ -126,6 +133,14 @@ async def type_label(self, info: Info) -> Optional[str]:
     async def stage_label(self, info: Info) -> str:
         return get_enum_label(StageTypeEnum, self.stage)
 
+    @strawberry.field
+    async def success_factor(
+        self, info: Info
+    ) -> Optional[List[SuccessFactorType]]:
```

Comment:
Let's return with empty array if empty
```suggestion
    ) -> List[SuccessFactorType]:
```